java/src/main/java/ai/rapids/cudf/HostColumnVector.java

```diff
@@ -738,15 +738,49 @@ public ColumnBuilder appendLists(List... inputLists) {
     public ColumnBuilder appendStructValues(List<StructData> inputList) {
       for (StructData structInput : inputList) {
         // one row
-        rows++;
         append(structInput);
       }
       return this;
     }
 
+    private void resizeDataBuffer(int length) {
```

Comment:
Could we get some java docs here, or rename `length`?  It is not clear that `length` is how many more bytes we need instead of how many bytes in total we would need.  Also because this only ever grows the data buffer `resize` might not be the best choice of a name.